src/pages/docs/custom-renderers.mdx

```diff
@@ -84,29 +84,34 @@ In this case, the props are `data`, which is the actual data bit to be rendered,
 a dot-separated string, which is necessary to propagate an update back to the store.
 
 For the `onClick` prop we pass the `handleChange` handler, which we retrieve
-via another helper function provided by JSON Forms called `withJsonFormsControlProps`.
+via another helper function provided by JSON Forms called `mapDispatchToControlProps`.
 All the handler actually does is to emit a change with the new value.
 
-The complete code of `src/RatingControl.js` looks as follows:
+The complete code of `src/RatingControl.tsx` looks as follows:
 
 ```jsx
 import * as React from 'react';
-import { 
-  mapStateToControlProps, 
-  mapDispatchToControlProps 
-} from '@jsonforms/core';
-import { withJsonFormsControlProps } from '@jsonforms/react'
-import { Rating } from './Rating';
+import {mapDispatchToControlProps, mapStateToControlProps} from '@jsonforms/core';
+import {connect} from 'react-redux';
+import {Rating} from './Rating';
+
+interface RatingControlProps {
+  data: any;
+  handleChange(path: string, value: any): void;
+  path: string
+}
 
-const RatingControl = 
-  ({ data, handleChange, path }) => (
+const RatingControl = ({ data, handleChange, path }: RatingControlProps) => (
   <Rating
     value={data}
-    onClick={ev => handleChange(path, Number(ev.value))}
+    onClick={(ev: any) => handleChange(path, Number(ev.value))}
   />
 );
 
-export default withJsonFormsControlProps(RatingControl);
+export default connect(
```

Comment:
Same here, keep `withJsonFormsControlProps`

Reply:
Reverted the changes.